### ISSUE TITLE
Would it be helpful to provide whether the app is ad-supported or supports in-app purchases?

### ISSUE THREAD

**Issue by aarongustafson** [w3c/manifest-app-info#8]
This is something many app stores flag-up for consumers.

**Comment by TomCJones** [comment]
Yes, this issue came up in the BlinkOn meeting today. I believe the answer to be -- definitely YES!  And further block attempts to change with user without some effort by the user to indicate that the user has changed her mind.  I am particularly interested in the PaymentRequest.
